website/docs/d/pi_instance.html.markdown

```diff
@@ -78,10 +78,12 @@ In addition to all argument reference list, you can access the following attribu
   Nested scheme for `networks`:
   - `external_ip` - (String) The external IP address of the instance.
   - `ip` - (String) The IP address of the instance.
-  - `macaddress` - (String) The MAC address of the instance. Deprecated please use `mac_address` instead.
```

Comment:
did we want to remove the deprecated mac address here from the documentation? It's gone here and in pi_instances.html.markdown but not in pi_instance_ip.html.markdown

Reply:
Removed